lib/stdlib/src/queue.erl

```diff
@@ -706,8 +706,8 @@ r2f([_]=R) ->
 r2f([X,Y]) ->
     {[X],[Y]};
```

Comment:
Sorry about pressing "Add single comment" too early; but it would be nice to swap `X` and `Y` here also...
It is fine to amend it to the previous rename commit.